tools/Equinox.Tools.TestHarness/Equinox.Tools.TestHarness.fsproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
I think readme has some remaining refs to netstandard which can be removed (not critical to do in this PR)

Reply:
sorted, thanks!